Riot/Assets/en.lproj/Vector.strings

```diff
@@ -84,10 +84,10 @@
 "auth_invalid_email" = "This doesn't look like a valid email address";
 "auth_invalid_phone" = "This doesn't look like a valid phone number";
 "auth_missing_password" = "Missing password";
-"auth_add_email_message" = "Add an email address to your account to let users discover you, and to reset your password.";
-"auth_add_phone_message" = "Add a phone number to your account to let users discover you.";
-"auth_add_email_phone_message" = "Add an email address and/or a phone number to your account to let users discover you. Email address will also let you reset your password.";
-"auth_add_email_and_phone_message" = "Add an email address and a phone number to your account to let users discover you. Email address will also let you reset your password.";
+"auth_add_email_message_2" = "Set an email for account recovery. Use email to optionally be discoverable by existing contacts.";
+"auth_add_phone_message_2" = "Set a phone for account recovery. Use phone to optionally be discoverable by existing contacts.";
```

Comment:
Also can a phone number be used for account recovery? I think it's not supported yet in Riot mobile app

Reply:
Afaik no. On web we avoided this copy for this reason.